supervisor/backups/backup.py

```diff
@@ -640,6 +640,22 @@ def _save() -> None:
             # Take backup
             _LOGGER.info("Backing up folder %s", name)
 
+            def is_excluded_by_filter(item_arcpath: PurePath) -> bool:
+                """Filter out bind mounts in folders being backed up."""
+                full_path = origin_dir / item_arcpath.relative_to(".")
```

Comment:
From what I can tell that `relative_to()` does nothing really, but it matches the `arcname` below.